src/main/java/org/osiam/configuration/JacksonConfig.java

```diff
@@ -0,0 +1,20 @@
+package org.osiam.configuration;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.osiam.resources.helper.UserDeserializer;
+import org.osiam.resources.scim.User;
+import org.springframework.boot.autoconfigure.web.WebMvcAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class JacksonConfig extends WebMvcAutoConfiguration {
```

Comment:
I tried. If the `@Configuration` is not a sub class of `WebMvcAutoConfiguration` the `@Bean` of the type `Module` doesn't get picked up.